src/test/java/subway/RestAssuredTest.java

```diff
@@ -15,9 +16,15 @@ public class RestAssuredTest {
     @DisplayName("구글 페이지 접근 테스트")
     @Test
     void accessGoogle() {
-        // TODO: 구글 페이지 요청 구현
-        ExtractableResponse<Response> response = null;
+        ExtractableResponse<Response> response =
+                RestAssured.
+                        when().
+                        get("https://google.com").
+                        then().
+                        statusCode(200).
+                        extract();
```

Comment:
```suggestion
    ExtractableResponse<Response> response =
        RestAssured
            .when()
            .get("https://google.com")
            .then()
            .statusCode(200)
            extract();
```
`.`을 앞으로 보내보는 건 어떨까요?